assets/i18n/en.toml

```diff
@@ -45,28 +45,28 @@ savingsTotalEnergy = "{total} kWh charged"
 tabTitle = "Live community"
 power = "Charging power"
 powerSub1 = "{activeClients} of {totalClients} participants"
-powerSub2 = "are charging right now"
+powerSub2 = "charging…"
 greenShare = "Solar share"
-greenShareSub1 = "power is covered by"
-greenShareSub2 = "PV & battery storage"
-greenEnergy = "Solar energy"
+greenShareSub1 = "power derived from"
+greenShareSub2 = "solar, and battery storage"
+greenEnergy = "Solar"
 greenEnergySub1 = "charged with evcc"
 greenEnergySub2 = "since October 2022"
 
 [footer.telemetry]
-optIn = "I want to contribute my data as well."
-optInMoreDetails = "More details are available {0}."
+optIn = "I want to contribute my data."
+optInMoreDetails = "More details at {0}."
 optInMoreDetailsLink = "here"
 optInSponsorship = "Sponsoring required."
 
 [footer.sponsor]
 titleSponsor = "You are a supporter"
 titleNoSponsor = "Support us"
-thanks = "Thanks for your support, {sponsor}! It helps us with the further development."
+thanks = "Thank you, {sponsor}! You have helped the development."
```

Comment:
```suggestion
thanks = "Thank you, {sponsor}. You have helped the development."
```

Reply:
Not 100% sure, but looks like to much German translation

"Thank you, {sponsor}. You do help development."

Just my 5 cent as recommendation.

Reply:
"It helps us with the further development." shouldn't have "the", and it is still strange.
"It helps further development of the app"?

Reply:
Yeahh, "you helped .. " sounds like 1 time event.

Official translation (also using Google, which is sometime tricky) would be => "You support the further development of the app."

Reply:
`You are helping` ? Need to somehow  emphasise continuous support and sponsoring 
I would not call evcc an app ?

Reply:
You may exchange "app" by "software", but to me a running code is called an app ;-)

Reply:
To me app is something you download and get to use right away :) Definitely not a case here.
I would not call it an app, and leave it as `evcc` then. 

Or simply:
`"Thank you, {sponsor}! Your contribution is vital for further development."`

Reply:
That made me think otherwise evcc would go out of business.
I don't like the exclamation point because I think it shouts at the user. More-so in other languages.

Reply:
> Something like ", and we hope you are inspired to entice others to do the same" might be a psychology-win.

That is too cheesy for me ;) Anyone who uses evcc understands it's benefits, they are self-explanatory, so no need to "motivate" _current_ users. We are already the best advertisers to everyone we know ;)


>That made me think otherwise evcc would go out of business.

evcc is entirely sponsor funded, so continuous sponsorship is vital.